bedrock/products/templatetags/misc.py

```diff
@@ -435,54 +487,55 @@ def relay_monthly_price(ctx, product=RELAY_PRODUCT, plan=RELAY_12_MONTH_PLAN, co
     In Template
     -----------
 
-        {{ relay_monthly_price( product='relay-email',
+        {{ relay_monthly_price_formatted( product='relay-email',
                              country_code=country_code,
                              lang=LANG,
                             ) }}
     """
 
-    available_plans = _relay_get_plans(country_code, lang, product)
-    selected_plan = available_plans.get(plan, RELAY_12_MONTH_PLAN)
-    price = selected_plan.get("price")
-    currency = selected_plan.get("currency")
-    currency_locale = lang.replace("-", "_")
-    amount = _format_currency(price, currency, currency_locale)
+    plan = "monthly" if plan == "monthly" else "monthly_when_yearly"
 
-    markup = f"{amount}"
-    return Markup(markup)
+    currency = _relay_get_currency(product, country_code, lang)
+
+    product = product.replace("relay-", "")
+    price = settings.RELAY_STRIPE_PLAN_DATA[product]["prices"][currency][plan]
+
+    price = Decimal(price)
+    return price
 
 
 @library.global_function
-@jinja2.pass_context
-def relay_total_price(ctx, product=RELAY_PRODUCT, plan=RELAY_12_MONTH_PLAN, country_code=None, lang=None):
+def relay_monthly_price_formatted(product=RELAY_PRODUCT, plan=RELAY_12_MONTH_PLAN, country_code=None, lang=None):
     """
-    Render a localized string displaying a Relay plan's total price.
+    Render a localized string displaying a Relay plan's monthly plan.
 
     Examples
     ========
 
     In Template
     -----------
 
-        {{ relay_total_price(country_code=country_code, lang=LANG, product='relay-email') }}
+        {{ relay_monthly_price_formatted( product='relay-email',
+                             country_code=country_code,
+                             lang=LANG,
+                            ) }}
     """
 
-    period = 12 if plan == RELAY_12_MONTH_PLAN else 1
-    available_plans = _relay_get_plans(country_code, lang, product)
-    selected_plan = available_plans.get(plan, RELAY_12_MONTH_PLAN)
-    price = float(selected_plan.get("price"))
-    total = price * period
-    currency = selected_plan.get("currency")
+    currency = _relay_get_currency(product, country_code, lang)
+    amount = relay_monthly_price(product, plan, country_code, lang)
     currency_locale = lang.replace("-", "_")
-    amount = _format_currency(total, currency, currency_locale)
+    # default to en_US if format_currency does not recognize the locale.
+    try:
+        price = format_currency(amount, currency, locale=currency_locale)
+    except UnknownLocaleError:
+        price = format_currency(amount, currency, locale="en_US")
 
-    markup = f"{amount}"
+    markup = f"{price}"
```

Comment:
`format_currency` returns a string, so you can skip the f-string here and just return `Markup(price)` if you wanted. Unless you need to add more strings to this but at the moment that's not happening.